src/EditorFeatures/Core/Implementation/GoToDefinition/GoToDefinitionCommandHandler.cs

```diff
@@ -34,10 +34,8 @@ public CommandState GetCommandState(GoToDefinitionCommandArgs args, Func<Command
 
         public void ExecuteCommand(GoToDefinitionCommandArgs args, Action nextHandler)
         {
-            var textBuffer = args.SubjectBuffer;
-
             var caretPos = args.TextView.GetCaretPoint(args.SubjectBuffer);
-            if (caretPos.HasValue && TryExecuteCommand(textBuffer.CurrentSnapshot, caretPos.Value))
+            if (caretPos.HasValue && TryExecuteCommand(args.SubjectBuffer.CurrentSnapshot, caretPos.Value))
```

Comment:
Why remove this variable? Why not just replace "args.SubjectBuffer" on the line above with "textBuffer"?